### PR TITLE
Test 0.7 and 1.0 but allow failures on 1.0 for now

### PR DESCRIPTION
**Pull request by iamed2**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,19 +5,32 @@ os:
...
 julia:
   - 0.6
+  - 0.7
+  - 1.0
   - nightly
 matrix:
   fast_finish: true
   allow_failures:
     - julia: nightly
+    - julia: 1.0
```
Comment (ararslan) [inline .travis.yml]: rip

File: test/runtests.jl [hunk 2]
```diff
@@ -7,8 +7,8 @@ using Compat.Distributed
 using Base: BufferStream
 using TestSetExtensions
 
-eval(Syslogs, Meta.parse("UDP_PORT = 8080"))
-eval(Syslogs, Meta.parse("TCP_PORT = 8080"))
+Base.eval(Syslogs, Meta.parse("UDP_PORT = 8080"))
```
Comment (ararslan) [inline test/runtests.jl]: Should be `Core.eval`

File: .travis.yml [hunk 3]
```diff
@@ -5,19 +5,31 @@ os:
   - osx
 julia:
   - 0.6
+  - 0.7
+  - 1.0
   - nightly
 matrix:
   fast_finish: true
   allow_failures:
     - julia: nightly
```
Comment (rofinn) [inline .travis.yml:14]: Forgot to add 1.0.

Comment (iamed2) [reply .travis.yml:14]: No I switched it on purpose after because I thought I could get it working; I'll check again later today

**Pull request closed without merging**